### PR TITLE
Fix URL in SUSHI-generated config file

### PR DESCRIPTION
**Pull request by masnick**
The URL that points to the SUSHI configuration docs is wrong in the generated configuration file. I think this will fix that, along with cleaning up other references to that.

### CODE REVIEW

**Comment by cmoesel** [comment]
Hey @ngfreiter -- looks like @masnick beat you to it!

**Comment by cmoesel** [comment]
Oh wait.  I thought you had done that already @ngfreiter.  Now I see you had in #744.  So... are there multiple paths to getting the wrong URL then?

**Comment by cmoesel** [comment]
Ohh....  I think @ngfreiter's PR fixed it for `sushi --init`, but this one fixes it for when we generate a config based on an existing `package.json` (or IG JSON? I forget which)...

File: src/utils/init-project/sushi-config.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ╭──────────────────────────────────────ImplementationGuide───────────────────────────────────────╮
 # │  The properties below are used to create the ImplementationGuide resource. For a list of       │
-# │  supported properties, see: https://fshschool.org/sushi/configuration/                         │
+# │  supported properties, see: https://fshschool.org/docs/sushi/configuration/                         │
```
Comment (jafeltra) [inline src/utils/init-project/sushi-config.yaml:3]: I don't believe this actually affects the tests in any meaningful way, but the new link is a few characters longer than the old, so the `|` that closes the box on this line is a little too far out to correctly form the box. Would you mind removing a few of the trailing spaces between the link and the `|` so the boxes lines up here? Also, if you wouldn't mind, would you be able to make that change in the other two test fixtures below this one?

**Review by jafeltra — COMMENTED** [review]
Thanks for the PR @masnick and for catching these extra incorrect links! I just had one small comment below. Let me know what you think!

File: src/utils/init-project/sushi-config.yaml (hunk 1 above)
Comment (cmoesel) [reply src/utils/init-project/sushi-config.yaml:3]: @masnick -- is this something you can quickly fix for us, or would you prefer we just pop in and do it?  I'm not sure if we have permission to push to your fork or not...

**Comment by cmoesel** [comment]
OK.  I've fixed the border alignment.  Ready for your review whenever, @jafeltra!

**Review by jafeltra — APPROVED** [review]
```
╭──────────────Approval───────────────╮
│  Borders are aligned to perfection. │ 
╰─────────────────────────────────────╯
```

**Comment by cmoesel** [comment]
```
╭──────────────Approval───────────────╮
│  Borders are perfect! │ 
╰─────────────────────────────────────╯
```
Doh!

**Pull request merged**